src/components/Entry.vue

```diff
@@ -179,7 +178,11 @@ export default {
       }
     },
     setURL() {
-      window.location.hash = "#" + this.eur.toString();
+      if (this.eur != 1) {
```

Comment:
```suggestion
      if (this.eur !== 1) {
```

Be (type-)safe out there, use strict checks.

Reply:
Werkt niet helaas. Als de waarde 1 of 1.00 is, toont hij hem alsnog in de URL balk.